Setup

DiGyt opened the pull request:
Add initial unity setup (with correct setting, but without mirror install) into workflow

tmhatton committed:
Adjusted gitattributes

tmhatton committed:
Initial Project setup commit

DiGyt committed:
added test changes

to see if everything works fine

tmhatton committed:
extended gitignore with ./idea

tmhatton committed:
Collapse Unity-generated files on GitHub

tmhatton commented:
closes #1 #2 #3 #4 #5

Hendrik1993 committed:
Szene mit neuer Lichtquelle

Hendrik1993 committed:
Änderungen beim öffnen von unity

Hendrik1993 committed:
Sphere statt lichtquelle

tmhatton commented:
@lucasessmann, entspricht wie Folder Structure ungefähr der von Westbrück? 
Wenn nicht könntest du dir diesen Branch noch ziehen, testen und dann die neue Folder Structure auch hier rein pushen? Danach sollten wir das eigentlich in workflow mergen können.

JasminLWalter committed:
Added WorkFlowScene and Room Objects

added a WorkFlowScene (with floor, shelf and some prototyp distractors & target)
added Prefabs folder
added Prefabs for Floor, Shelf, Prototyp Distractors, Prototyp Target, Material for Prototyp Distractors/Targeta

lucasessmann committed:
General Folder structure - need to sort and think about it further though
